CMSgov/design-system
# [WNMGDS-1142] Clean up files and READMEs for ds-medicare-gov

https://jira.cms.gov/browse/WNMGDS-1142

## Summary
For context, see https://github.com/CMSgov/design-system/pull/1212

### Changed
- Delete unnecessary files
- Update READMEs

## How to test
Take a look at the READMEs and make sure the commands to build and test still work.
